client/state/sites/plans/actions.js

```diff
@@ -123,6 +125,21 @@ export function fetchSitePlans( siteId ) {
  */
 export function fetchSitePlansCompleted( siteId, data ) {
 	data = omit( data, '_headers' );
+	const { product_id } = personalPlan;
+
+	if ( ! isEmpty( data ) && ! data.hasOwnProperty( product_id ) ) {
```

Comment:
we know that if `data` has `product_id`, it also won't be empty, so the duplicate check here is redundant. using something like **lodash**'s `has` will also provide safety against `data` being `undefined`

```js
import has from 'lodash/has';

if ( has( data, 'product_id' ) ) {
	…
}
```